.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,35 @@
+name: dependabot
+
+on:
+  pull_request:
+    branches:
+      - dependabot/**
+  push:
+    branches:
+      - dependabot/**
+  workflow_dispatch:
+
+permissions:
+  contents: write # Allow to update the PR.
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # tag=v3.5.0
+      with:
+        go-version: '=1.20.7'
+      id: go
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # tag=v3.3.0
+    - name: Update generated code
+      run: make generate
```

Comment:
perhaps we could also run go mod tidy https://github.com/rancher-sandbox/rancher-turtles/blob/main/Makefile#L224-L227?

Reply:
It is already a subtask of `make generate` [here](https://github.com/rancher-sandbox/rancher-turtles/blob/main/Makefile#L202)